blocks/filesystem.py

```diff
@@ -190,7 +226,7 @@ def access(self, paths):
         """
         # Move the files into a tempdir from GCS
         tmpdir = _session_tempdir()
-        self.copy(paths, tmpdir)
+        self.cp(paths, tmpdir)
```

Comment:
Are you deleting the files in tmpdir after the copy operation?

Reply:
Yes the _session_tempdir cleans up after itself at the end of program execution